logstash-core/src/main/java/org/logstash/execution/AbstractPipelineExt.java

```diff
@@ -190,6 +191,15 @@ public void notify(ConditionalEvaluationError err) {
             } catch (IOException ioex) {
                 LOGGER.warn("Invalid operation on closing internal resources", ioex);
             }
+
+            // if pipeline has DLQ enabled, send also there the event
+            if (javaDlqWriter != null) {
```

Comment:
We should change the log entry to mention that the event was sent to the DLQ instead of "Event was dropped, enable debug logging to see the event's payload."